### DISCUSSION TITLE
handling pug in webpack setupMiddlewares

### DISCUSSION THREAD

**Discussion by exocornet** [webdiscus/pug-plugin#94] [Q&A]
Hello. Please tell me. How can I handle pug in webpack setupMiddlewares using your plugin?

![pug-loader](https://github.com/webdiscus/pug-plugin/assets/57918887/d12463e0-1aba-44e5-b3e7-3c565a082f08)

**Comment by webdiscus** [comment]
Hello @exocornet,

 PugPlugin does not support the use of middleware, sorry.
